### PR TITLE
Add median to paddle.tensor.stat

### PR DESCRIPTION
**Pull request by SANTHOSHKAKARLA**
Close https://github.com/unifyai/ivy/issues/15774

### CODE REVIEW

**Comment by SANTHOSHKAKARLA** [comment]
Close #15774

**Comment by KSANTHOSH200** [comment]
Hi @Aarsh2001,

Can you please review this PR?

Thanks.

**Comment by SANTHOSHKAKARLA** [comment]
Hi @Aarsh2001,

Can you please review this PR?

Thanks.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_stat.py [hunk 1]
```diff
@@ -34,3 +34,30 @@ def test_paddle_mean(
         axis=axis,
         keepdims=keepdims,
     )
+    
+# median
+@handle_frontend_test(
+    fn_tree="paddle.median",
+    dtype_and_x=statistical_dtype_values(function="median"),
```
Comment (Aarsh2001) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_stat.py:41]: the function is `_statistical_dtype_values`

**Review by Aarsh2001 — CHANGES_REQUESTED** [review]

**Comment by Aarsh2001** [comment]
closing this PR due to inactivity, feel free to reopen once you're ready with your changes 🙂

**Pull request closed without merging**